### DISCUSSION TITLE
Pleccer (PrusaSlicer flavour) v. Alpha 4 released

### DISCUSSION THREAD

**Discussion by rvmn** [rvmn/SuperPleccer#5] [Announcements]
Now implemented in final way using PrusaSlicer as base.
See the [release page](https://github.com/rvmn/SuperPleccer/releases/tag/alpha4_prerelease) for more info.

**Comment by 5axes** [comment]
It's hard to understand, we are on a version of SuperSlicer, PrusaSlicer version 2.6 Alpha 4 but with the Functions of SuperSlicer 2.5 ? Or just Prusa 2.6 Alpha 4 ?   It would be worthwhile to have some clarification on the level of Functions integrated in the Slicer.

**Reply by rvmn** [reply]
apologies for the weird release on a repo that is a superslicer fork.
 ### Pleccer is now based on prusa 2.6a4, there will not be a superslicer version
### i will if people would like this do a PR on superslicer with the arc infill, but only in a few weeks when it's finished and tested. Expect a poll for on this repo in about 2 weeks, so please subscribe and be noticed.

btw, this decision was made because of prusa having a.o. organic supports and arachne.

**Comment by 5axes** [comment]
Thank you for this feedback. It would also be interesting to integrate in the versions the possibility to activate the different languages. Even if the translation is not complete (Especially if the translation is not complete) It allows to see what are the parameters added compare to a standart release (those labels which remain in English compared to the chosen language). Currently in the Alpha version only English is available even if the language files are well present in the **_Localization_** folder.

![image](https://user-images.githubusercontent.com/11015345/222209553-e388e553-71f9-4b31-ad87-7d2fa46dee8a.png)

**Comment by radarOverhead** [comment]
Tries to run this new version on Linux and getting an error on startup.

Pleccer-alpha Error

Internal error: boost::filesystem::status: Permission denied [system:13]:
"/root/PrusaSlicer/deps/build/destdir././/local/ssl/cert.pem"

Any idea ?  Apologies if this is something I should know how to fix.

**Reply by pleccer** [reply]
It's linking to a file on the VPS where i compiled it. that shouldn't be hardlinked, arrr.. :-/
I think it probably is confused and in a sort of identity crisis!
Try checking whether you have a folder like /home/*username*/.config/Pleccer-alpha/ and make that folder empty.
That should reset it's identity.

**Comment by patrezp** [comment]
Looks like the Alpha4 isn't working on the ARM M2 Mac, tried both dmg and zip
<img width="260" alt="image" src="https://user-images.githubusercontent.com/743265/223124541-c0dd70cd-9717-4a75-814e-b075671919df.png">

**Reply by rvmn** [reply]
Can you try again with the new arm file i uploaded to the release page? 
Forget about the normal dmg, that is for Intel Mac.

**Reply by pleccer** [reply]
If that doesn't work, could you try getting a superslicer, taking the files out and replacing the files by pleccer files and test if that does work? Then i know packaging fixes the problem.

**Reply by pleccer** [reply]
https://github.com/supermerill/SuperSlicer/releases/tag/2.4.58.5

**Reply by patrezp** [reply]
Will try later on today and let you know.
By taking the files out you mean open the app folder of superslicer and overwrite the contents with contents of superpleccer zip?

**Reply by pleccer** [reply]
exactly, thanks in advance!

**Reply by patrezp** [reply]
So that did not help.  I tried to use Superslicer executable renamed to Pleccer, based on the Info.plist, since your zip does not contain any executable
<img width="260" alt="image" src="https://user-images.githubusercontent.com/743265/223422996-7bc4dfb9-ef5f-4c82-a1f8-571f6aac28fe.png">

<img width="800" alt="image" src="https://user-images.githubusercontent.com/743265/223423339-a66d68a1-793c-452e-a971-ce44d4d70143.png">

**Reply by rvmn** [reply]
@patrezp An ARM Mac version is not compiling (yet), but in the release page you now find a Intel mac version and you can use that instead.

**Reply by rvmn** [reply]
> @patrezp An ARM Mac version is not compiling (yet), but in the release page you now find a Intel mac version and you can use that instead.

I tested in a VM (Docker-OSX) and it runs fine, but apparently you do need to do a `brew install zstd` (with Homebrew installed). 
Then for ARM Mac to allow running the app you also need to extra thing on the commandline: `xattr -cr ./PrusaSlicer.app`

**Comment by gettingerr** [comment]
Should I be seeing Arc as an infill option? Trying to do some testing with arc overhangs and it seems the Alpha4 version is meant to support them but cannot find the setting. (On windows)

**Reply by rvmn** [reply]
Sorry, this may be a bit unclear, in this version all overhangs are using arc infill, there is no option to turn it off (will be there later).

Also take into account a few things:
1) On bigger overhangs use paint-on-supports and paint a line to support the overhang
2) Auto supports will by default be underneath the perimeters, which will be printed last in overhangs, it maybe better to be printing perimeters first (changeable with 'overhang infill first' option)
3) As said in release info make sure in multi object jobs to split the objects (icon in top menu)

**Reply by gettingerr** [reply]
That all makes perfect sense! Thank you for the response and clear instructions I will try it out! And thank you for the hard work putting it together

**Comment by CodyPrince** [comment]
Very impressed with Pleccer. Keep up the good work.

Hoping to see if your next beta will use Pursaslicer aplha 5. Theyve done well with their Extend sparse infill and Dynamic control of fan speed on overhangs update over alpha 4.

Cody
